README

```diff
@@ -30,7 +30,7 @@ Prerequests
 3. Documentation (optional)
    * asciidoctor (http://asciidoctor.org/) available as `asciidoctor`
    * If not present, docs are not formatted, but left in readable ASCII
-   * Also available on-line at http://nanomsg.org/documentation
+   * Also available online at http://nanomsg.org/documentation
```

Comment:
Huh.  I prefer with the dash, but I'm stodgy apparently:  http://grammarist.com/spelling/online-on-line/